tox.ini

```diff
@@ -1,18 +1,20 @@
 [tox]
-minversion = 2.1
-envlist = py{27,35,36}-django{19,110,111}
+minversion = 3.1
+envlist = py{36,37,38}-django{2x,20,21}{,-faster}
```

Comment:
Unimportant:

```suggestion
envlist = py{36,37,38}-django{2x,20,21}-{,faster}
```

Reply:
Is that correct?
I would expect the outputs of  `django20{,-faster}` to be `django20` and `django20-faster`
and the outputs of `django20-{,faster}` to be `django20-` and `django20-faster`

I also haven't used tox in a while so I may be misunderstanding what the expectations are.

Reply:
You're correct

```
~/projects/forks/django-include django2
❯ tox -l
py36-django2x
py36-django2x-faster
py36-django20
py36-django20-faster
py36-django21
py36-django21-faster
py37-django2x
py37-django2x-faster
py37-django20
py37-django20-faster
py37-django21
py37-django21-faster
py38-django2x
py38-django2x-faster
py38-django20
py38-django20-faster
py38-django21
py38-django21-faster

~/projects/forks/django-include django2
❯ tox -l
py36-django2x-
py36-django2x-faster
py36-django20-
py36-django20-faster
py36-django21-
py36-django21-faster
py37-django2x-
py37-django2x-faster
py37-django20-
py37-django20-faster
py37-django21-
py37-django21-faster
py38-django2x-
py38-django2x-faster
py38-django20-
py38-django20-faster
py38-django21-
py38-django21-faster
```

but tox just ignores the trailing `-` as a noop.

That said, I think what you have here is better. It's weird seeing the trailing `-` in the `tox -l`. Disregard my suggestion